src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/TransformFunctions.kt

```diff
@@ -512,70 +506,6 @@ private fun extractDates(sentence: CalculableSentence): Map<ReleaseDateType, Dat
   return dates
 }
 
-fun transform(booking: Booking): SentenceDiagram {
```

Comment:
some remnants of the SentenceDiagram functionality that has been removed from the main code base